### PR TITLE
Change some rough/inappropriate language in comments

### PR DESCRIPTION
**Pull request by random-geek**
It doesn't add anything, and it doesn't seem appropriate for an open-source project, especially one of this size and with so much appeal to children.

### CODE REVIEW

File: src/map.cpp [hunk 1]
```diff
@@ -179,7 +179,7 @@ void Map::setNode(v3s16 p, MapNode & n)
 	v3s16 blockpos = getNodeBlockPos(p);
 	MapBlock *block = getBlockNoCreate(blockpos);
 	v3s16 relpos = p - blockpos*MAP_BLOCKSIZE;
-	// Never allow placing CONTENT_IGNORE, it fucks up stuff
```
Comment (MoNTE48) [inline src/map.cpp]: 😂
I agree with the comment, although such things are in the source code...

**Comment by paramat** [comment]
:+1:

File: src/gui/guiEditBoxWithScrollbar.cpp [hunk 2]
```diff
@@ -1112,7 +1112,7 @@ void GUIEditBoxWithScrollBar::breakText()
 // TODO: that function does interpret VAlign according to line-index (indexed line is placed on top-center-bottom)
 // but HAlign according to line-width (pixels) and not by row.
 // Intuitively I suppose HAlign handling is better as VScrollPos should handle the line-scrolling.
-// But please no one change this without also rewriting (and this time fucking testing!!!) autoscrolling (I noticed this when fixing the old autoscrolling).
+// But please no one change this without also rewriting (and this time testing!!!) autoscrolling (I noticed this when fixing the old autoscrolling).
```
Comment (paramat) [inline src/gui/guiEditBoxWithScrollbar.cpp]: Wrap all comment lines at 80-90 columns (tab size 4)

**Comment by paramat** [comment]
:+1:

**Comment by sfan5** [comment]
I question the usefulness and necessity of this change, especially since code comments are not user-facing documentation.
Probably not worth arguing over, though.

**Comment by paramat** [comment]
Well, it is unlikely that the engine code will be worked on by younger students in schools, but it might, and i have had teachers tell me they could be in serious trouble in this situation. I would rather not risk it.
I quite like c55s classic 'fruity' comments, but the recently added F-word rant is unacceptable and unpleasant.

**Pull request merged**